Examples/Swift/LiveDataExample.swift

```diff
@@ -0,0 +1,49 @@
+import Mapbox
+
+@objc(LiveDataExample_Swift)
+
+class LiveDataExample: UIViewController, MGLMapViewDelegate {
+    
+    var source : MGLShapeSource!
+    var timer = Timer()
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let mapView = MGLMapView(frame: view.bounds,
+                                 styleURL: MGLStyle.darkStyleURL(withVersion: 9))
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.delegate = self
+        
+        mapView.tintColor = .gray
+        view.addSubview(mapView)
+    }
+    
+    func mapView(_ mapView: MGLMapView, didFinishLoading style: MGLStyle)
+    {
+        if let url = URL(string: "https://wanderdrone.appspot.com/") {
+            // Add a source to the map. https://wanderdrone.appspot.com/ generates coordinates for simulated paths.
+            source = MGLShapeSource(identifier: "wanderdrone", url: url, options: nil)
+            style.addSource(source)
+            
+            // Add an icon to the map to represent the drone's coordinate.
+            let droneLayer = MGLSymbolStyleLayer(identifier: "wanderdrone-layer", source: source)
+            droneLayer.iconImageName = MGLStyleValue(rawValue: "rocket-15")
```

Comment:
Same as in the Objective-C example, lets clarify where that rocket icon is coming from.